config/jobs/kubernetes/cloud-provider-vsphere/cloud-provider-vsphere-config.yaml

```diff
@@ -5,13 +5,19 @@ presets:
     preset-cloud-provider-vsphere-e2e-config: "true"
   env:
   - name: CONFIG_ENV
-    value: /root/.cloud-provider-vsphere/config.env
+    valueFrom:
+      secretKeyRef:
+        name: cloud-provider-vsphere-ci
+        key: CONFIG_ENV
```

Comment:
This would be taking the value of `CONFIG_ENV` and putting it directly in the environment variable, as opposed to a file name.. is this intended, as you're also still mounting the secret to the filesystem down on L20

Reply:
Yeah, I made a mistake there, fixed it, thanks @cblecker !